spec/models/event_spec.rb

```diff
@@ -25,6 +25,21 @@
     end
   end
 
+  context "when author does not take part in project" do
+    subject { FactoryGirl.build(:event) }
+    before { subject.author = FactoryGirl.build(:user) }
+
+    it "is invalid" do
+      expect(subject).to be_invalid
+    end
+
+    it "adds error message about author" do
+      subject.validate
+      expect(subject.errors.messages[:author].first).
+        to eql("must take part in project")
```

Comment:
line 39 is indented further because it's method invocation chained to object in previous line